### PR TITLE
Review downgrade controller

### PR DESCRIPTION
**Pull request by behica**

### CODE REVIEW

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -30,4 +33,18 @@ def create
...
       redirect_to new_charge_path
   end
+  
+  def destroy
+    Stripe.api_key = Rails.configuration.stripe[:secret_key]
+    customer = Stripe::Customer.retrieve(current_user.customer_id)
+    sub = customer.subscriptions.first
+    sub.delete
+    
+    current_user.update(role: 0)
```
Comment (bobbilee19) [inline app/controllers/charges_controller.rb:43]: u can do update current_user with both things in one line, yah

File: app/controllers/charges_controller.rb [hunk 2]
```diff
@@ -21,6 +22,8 @@ def create
       plan: 'premium_membership'
     )
     
+    current_user.update(customer_id: customer.id)
+    # current_user.update(subscription_id: id)
```
Comment (bobbilee19) [inline app/controllers/charges_controller.rb]: just delete comments if not using code, if for u to reference i would just put comment in the commit on github

Comment (behica) [reply app/controllers/charges_controller.rb]: Yeah whoops, meant to take it out but forgot to save before committing. Took it out in the next branch. Good catch.

File: app/controllers/charges_controller.rb (hunk 1 above)
Comment (behica) [reply app/controllers/charges_controller.rb:43]: Do I do that with just separating them with commas? Like `current_user.update(role: 0, customer_id: customer.id)`?

Comment (behica) [reply app/controllers/charges_controller.rb:43]: Also, is `update` fine or should I be using `update_attributes` then?

**Pull request merged**